src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service;
+
+import static java.nio.file.Files.readAllLines;
+import static org.junit.Assert.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static final String FILE_INPUT = "src/main/resources/allTransactionForPeriod.csv";
```

Comment:
Your tests should not read files from 'main' package. You should create folder 'resources' under 'test' for only test purposes.
```suggestion
    private static final String FILE_INPUT = "src/test/resources/allTransactionForPeriod.csv";
```
![image](https://user-images.githubusercontent.com/117547096/213883471-43fa3133-77d5-42ee-945e-413838dee721.png)